docs/sources/setup-grafana/configure-security/configure-scim-provisioning/manage-users-teams/_index.md

```diff
@@ -0,0 +1,160 @@
+---
+description: Learn how to implement SCIM provisioning in Grafana for automated user and team synchronization. SCIM integrates with identity providers like Okta and Azure AD to streamline user management, automate team provisioning, and replace Team Sync.
+keywords:
+  - grafana
+  - scim
+  - provisioning
+  - user-management
+  - team-management
+labels:
+  products:
+    - cloud
+    - enterprise
+menuTitle: Manage users and teams with SCIM
+title: Manage users and teams with SCIM
+weight: 310
+---
+
+# Manage users and teams with SCIM
+
+{{< admonition type="note" >}}
+Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+{{< /admonition >}}
+
+SCIM streamlines identity management in Grafana by automating user lifecycle and team membership operations. This guide explains how SCIM works with existing Grafana setups, handles user provisioning, and manages team synchronization.
+
+With SCIM, you can:
+
+- **Automate user lifecycle** from creation to deactivation
+- **Manage existing users** by linking them with identity provider identities
+- **Synchronize team memberships** based on identity provider group assignments
```

Comment:
Automate team lifecycle from creation to deactivation - would be good to call out the auto-creation and deletion of team?